wiki/People/The_Team/Beatmap_Nominators/Rules/en.md

```diff
@@ -22,7 +22,7 @@ Beatmap Nominators possess the tools to nominate beatmaps and shape the world of
 
 ### Qualification
 
-- **Do not qualify beatmapsets whose song already has a Qualified beatmapset, unless no game modes of both sets overlap—this includes remixed versions**. For example an osu!taiko and osu! beatmapset of the same song can exist in the Qualified section, whilst an osu!taiko and osu!/osu!taiko hybrid set cannot. This ensures variety among Ranked beatmaps within their respective game modes.
+- **Only one beatmap of a song may be qualified per game mode at a time. This includes different length cuts and simple covers of a song, but excludes remixes, up to three different versions of which may be qualified per game mode at a time.** This ensures variety among Ranked beatmaps within their respective game modes.
```

Comment:
- should use similar language as the other points, speaking directly to the bn. so first sentence could be "Do not qualify beatmapsets with songs already used in Qualified beatmaps of the same mode." ?
- "includes" and "excludes" could be interpreted to mean the opposite of your intention, I think that's why the original was confusing. "Cuts and covers do not count as different songs when applying this rule, but remixes do" or somethign similar that sounds better for second sentence 
- possibly remove the remixes thing altogether per The Dm
- can drop the "within their respective game modes" from last sentence cuz its obvious and just puts morewords in already wordy rule

Reply:
reworded